spec/helpers/action_view_extension_spec.rb

```diff
@@ -74,6 +74,31 @@
     it { is_expected.not_to include "00" }
   end
 
+  describe '#money_only_cents' do
+    let(:monetizable_object){ Money.new(125_00) }
+    subject { helper.money_only_cents monetizable_object }
+    it { is_expected.to be_a String }
+    it { is_expected.not_to include Money.default_currency.decimal_mark }
+    it { is_expected.not_to include Money.default_currency.symbol }
+    it { is_expected.to include "00" }
```

Comment:
This should be `is_expected.to eq "00"` (instead of `include`), otherwise result of `12500` would also pass this spec

Reply:
good catch, thanks